### ISSUE TITLE
Osgloglas is insanely bugged

### ISSUE THREAD

**Issue by Poochyh** [xJon/The-1.12.2-Pack#676]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
**yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Information required**
1. Is the modpack crashing: **no**
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)):
3. Is Optifine installed or any other additional mods: **no**

**Describe the issue**
A clear and concise description of what the issue is.

Osgloglas refuses to be cast into anything. You can't cast it into a block, ingot or into any tool part. Osgloglas only pours in 6mb per right click onto a faucet and then stops, it registers the 6 mb as a full ingot, so temporarily you lose an ingot in the smeltery, but if you wait a bit or break the casting table/basin, the osgloglas magically reappears in the smeltery. I've tried to cast it by first pouring it into a seared tank, didn't work, then I tried a thermal expansion portable tank, that didn't work either. There is a way to pour it into a cast or a block, you need to spam right click on the faucet, but even if you get the cast full of osgloglas, it'll go up to 100% complete/cooled but it'll stay there forever and you're able to remove the cast while it's still cooling which to my knowledge shouldn't be possible in tinkers' construct. You can't use it on a channel either, the osgloglas gets stuck in the channel block with the max 32 mb if you're trying to pour it into a cast/basin with the channel.

**Comment by xJon** [comment]
Odd, I can't seem to reproduce the issue you're describing. Are you playing on a singleplayer world or on a server?
![Screenshot_1](https://user-images.githubusercontent.com/8945347/116882425-1cb40e00-ac2d-11eb-95c5-3fcb7e07db51.png)

**Comment by Poochyh** [comment]
I play on a multiplayer server called Apocolypse Gaming Network. I have a suspicion that it may be due to server lag since I've encountered other ores sometimes behaving like that, but I just broke the casting table and tried again and it worked for them. Osgloglas just seems a lot more resilient. I'll try tomorrow to build a separate smeltery to see if the issue is in that.  But it's good to know that the issue isn't in the modpack, thank you for your assistance.
